pubspec.yaml

```
@@ -59,7 +64,10 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
+  assets:
+      - assets/images/ 
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)